src/pages/patch/patchTabs/DownstreamProjectAccordion.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { Accordion } from "components/Accordion";
+import { PatchStatusBadge } from "components/PatchStatusBadge";
+import { useToastContext } from "context/toast";
+import { PatchTasksQuery, PatchTasksQueryVariables } from "gql/generated/types";
+import { GET_PATCH_TASKS } from "gql/queries";
+import { useNetworkStatus } from "hooks";
+import { PatchTasksTable } from "./tasks/PatchTasksTable";
+
+interface DownstreamProjectAccordionProps {
+  projectName: string;
+  status: string;
+  taskCount: number;
+  childPatchId: string;
+}
+export const DownstreamProjectAccordion: React.FC<DownstreamProjectAccordionProps> = ({
+  projectName,
+  childPatchId,
+  status,
+}) => {
+  const dispatchToast = useToastContext();
+
+  const { data, startPolling, stopPolling } = useQuery<
+    PatchTasksQuery,
+    PatchTasksQueryVariables
+  >(GET_PATCH_TASKS, {
+    variables: {
+      patchId: childPatchId,
+    },
+    fetchPolicy: "cache-and-network",
+    onError: (err) => {
+      dispatchToast.error(`Error fetching downstream tasks ${err}`);
+    },
+  });
+  const showSkeleton = !data;
+  useNetworkStatus(startPolling, stopPolling);
+  const { patchTasks } = data || {};
+
+  const variantTitle = (
+    <>
+      <ProjectTitleWrapper>
+        <span data-cy="project-title">{projectName}</span>
+      </ProjectTitleWrapper>
+      <PatchStatusBadge status={status} />
+    </>
+  );
+  return (
+    <AccordionWrapper data-cy="project-accordion">
+      <Accordion
+        title={variantTitle}
+        contents={
+          <TableWrapper>
+            {/* todo: add pagination and filtering  */}
+            {showSkeleton ? (
+              <Skeleton active title={false} paragraph={{ rows: 8 }} />
+            ) : (
+              <PatchTasksTable sorts={[]} patchTasks={patchTasks} />
```

Comment:
We should use the TasksTable Component instead.
 https://github.com/evergreen-ci/spruce/blob/90b029f2a5adb667989261ae87f11406e41efe87/src/components/Table/TasksTable.tsx

Reply:
PatchTasksTable has some handling that is specific to the tasks table on the patch page. including things like updating query params and other stuff. It might make sense to build something off the TasksTable component that includes managing the sort state for each table. since there can be several on the screen at once

Reply:
Thanks, I'll look into that tomorrow.

Reply:
done